public/css/main.css

```diff
@@ -0,0 +1,934 @@
+  -webkit-align-items: stretch;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: flex-start;
+  align-items: stretch;
+}
+.flex-paragraph-1 {
+  -webkit-flex-grow: 1;
+  -webkit-flex-shrink: 0;
+  -webkit-flex-basis: 98%;
+  flex-grow: 1;
+  flex-shrink: 0;
+  flex-basis: 98%;
+}
+.flex-paragraph-2, .flex-paragraph-3 {
+  -webkit-flex-grow: 1;
+  -webkit-flex-shrink: 0;
+  -webkit-flex-basis: 48%;
+  flex-grow: 1;
+  flex-shrink: 0;
+  flex-basis: 48%;
+  padding-right: 1em;
+}
+.strengths {
+  background-color: #fd6632;
+  color: white;
+  font-family: Helvetica;
+  text-align: center;
+}
+.strengths figcaption {
+  font-size: 0.75em;
+  line-height: 1.5;
+  letter-spacing: 0.02em;
+  width: 80%;
+  margin: 1em auto 2.5em auto;
+}
+.icon-quill:before, .icon-flask:before, .icon-gear:before  {
+  display: inline-block;
+  width: 2em;
+  height: 2em;
+  border-radius: 50%;
+  background-color: #fff;
+  font-size: 3.157em;
+  color: #fd6632;
+  line-height: 2;
+}
+/*Pricing tables */
+table {
+  text-align: center;
+  background-color: #fff;
+  display: inline-block;
+  margin: 1em;
+}
+caption {
+  background-color: #7f3319;
+  padding: 1em;
+}
+th, td {
+  color: #414141;
+  padding: 1em;
+  border-top: 1px solid #fd6632;
+  font-size: 0.75em;
+}
+th strong {
+  font-size: 1.333em;
+}
+.price {
+  color: #fd6632;
+}
+.signup {
+  background-color: #be4c25;
+  padding: 1.5em 4.157em;
+  border-top: 1px solid #fd6632;
+}
+/* Services article desktop layout */
+@media screen and (min-width: 50em) {
+  .services {
+    padding-top: 3em;
+    padding-bottom: 3em;
+  }
+  .selling-points {
+    margin: 2em auto;
+  }
+  .strengths {
+    display: inline-block;
+    width: 32%;
+  }
+  .strengths figcaption {
+    font-size: 0.9em;
+  }
+  .price-tables h3 {
+    margin-bottom: 0.5em;
+  }
+
+}
+/*&&&&&&&&&&&&&&&&
+
+    Portfolio article
+
+&&&&&&&&&&&&&&&&&&*/
+.portfolio {
+  text-align: center;
+  color: rgb(124,131,135);
+  padding-top: 2.5em; 
+}
+.portfolio > img {
+  margin: 1em calc(50% - 470px); /* 470 is half the img width */
+}
+.secondary-nav ul {
+  list-style-type: none;
+  width: 100%;
+}
+.secondary-nav li {
+  box-sizing: border-box;
+  text-align: center;
+  width: 100%;
+  padding: .5em;
+  margin: 0.5em auto;
+  border: 2px solid rgba(0,0,0,0.2);
+  border-radius: 0.5em;
+}
+.portfolio h4 {
+  font-weight: bold;
+  color: #000;
+  padding-bottom: 0.5em;
+}
+.portfolio figure {
+  margin: 2em calc(50% - 150px) 1em calc(50% - 150px); /* 150px is half the img width */
+}
+.portfolio figcaption {
+  margin-top: 1em;
+}
+.portfolio figure a {
+  display: inline-block;
+  width: 300px;  /* dimensions of img */
+  height: 220px;
+  position: relative;
+}
+/* defines hover area for fadeout */
+.portfolio figure a:before {
+  content: "";
+  position: absolute;
+  display: inline-block;
+  top: 0;
+  left: 0;
+  height: 220px;  /* dimensions of img */
+  width: 300px;
```

Comment:
When specifying a full-coverage overlay like this, take advantage of the absolute positioning you already have in place; use `bottom: 0` and `right: 0` to make it occupy the entirety of it's parent element:

```
.portfolio figure a:before {
    content: "";
    position: absolute;
    display: inline-block;
    top: 0;
    left: 0;
    /* height: 220px; */
    /* width: 300px; */
    bottom: 0;
    right: 0;
}
```

What happens if someone comes along and uses a different sized image? Avoid using any absolute pixel dimensions you don't... absolutely... need to use.